api/handler/object_list.go

```diff
@@ -199,6 +200,15 @@ func fillContents(src []*data.ObjectInfo, encode string, fetchOwner bool) []Obje
 			ETag:         obj.HashSum,
 		}
 
+		if size, ok := obj.Headers[layer.AttributeDecryptedSize]; ok {
+			sz, err := strconv.ParseInt(size, 10, 64)
+			if err != nil {
+				zap.L().Warn("Failed to parse size from header", zap.String("header_value", size), zap.Error(err))
```

Comment:
It's an error. It should be returned in some way.

Reply:
Returned